addon.py

```diff
@@ -3,46 +3,39 @@
 
 from __future__ import absolute_import, division, unicode_literals
 import sys
-from routing import Plugin
 from inputstreamhelper import ADDON, Helper
 
-# NOTE: Work around an issue in script.module.routing
-#       https://github.com/tamland/kodi-plugin-routing/pull/16
-if len(sys.argv) < 1:
-    sys.argv.append('addon.py')
-if len(sys.argv) < 2:
-    sys.argv.append('-1')
 
-plugin = Plugin()
+def run(params):
+    ''' Route to API method '''
+    if len(params) >= 2:
+        if params[1] == 'widevine_install':
+            widevine_install()
+        elif params[1] == 'widevine_remove':
+            widevine_remove()
+        elif params[1] == 'check_inputstream':
+            if len(params) == 3:
+                check_inputstream(params[2])
+            elif len(params) == 4:
+                check_inputstream(params[2], drm=params[3])
```

Comment:
If there are more than 4 arguments specified or the first is neither of the 3, it would just do nothing. I think adding/logging some error like "invalid API call" would make sense.

Reply:
Thanks for noticing, I edited my pull request accordingly.